### ISSUE TITLE
PGAdmin image upgrade resulted in (13)Permission denied: AH00091: httpd: could not open error log file

### ISSUE THREAD

**Issue by mzwettler2** [CrunchyData/postgres-operator#3846]
I upgraded PGO 5.3.0 to 5.5.0. 
PGAdmin was implicitly upgraded from image "crunchy-pgadmin4:ubi8-4.30-8" to image "crunchy-pgadmin4:ubi8-4.30-19".
This resulted in the following error:

Defaulted container "pgadmin" out of: pgadmin, pgadmin-startup (init), nss-wrapper-init (init)
Mi Feb 14 12:36:32 Europe 2024 INFO: TLS disabled. Applying http configuration..
Mi Feb 14 12:36:32 Europe 2024 INFO: Starting Apache web server..
[Wed Feb 14 12:36:32.599923 2024] [so:warn] [pid 18:tid 140141756648896] AH01574: module wsgi_module is already loaded, skipping
(13)Permission denied: AH00091: httpd: could not open error log file /var/lib/pgadmin/error_log.
AH00015: Unable to open logs

The error disappeared when I downgraded the image back to "crunchy-pgadmin4:ubi8-4.30-8".

**Comment by tjmoore4** [comment]
Hello @mzwettler2. Sorry to hear you're hitting this issue. I did a quick upgrade test, and I was able to use pgAdmin and didn't notice that error after the upgrade. Did everything fully roll out after the upgrade in your environment? I'd suggest looking through your Pod logs to see if anything else may not have deployed as expected. Also, I should note that the [namespace scope pgAdmin implementation](https://access.crunchydata.com/documentation/postgres-operator/latest/architecture/namespace-scoped-pgadmin-4) has many improvements and is available starting in v5.5.0 so I'd recommend checking that out as well. Hope this helps!
